src/webextension/firstrun/components/app.js

```diff
@@ -4,11 +4,21 @@
 
 import React from "react";
 
-import Pages from "./pages";
-import Wizard from "./wizard";
+import Welcome from "./welcome";
+import MasterPasswordSetup from "./master-password-setup";
+
+import styles from "./app.css";
 
 export default function App() {
   // Eventually, we'll have a feedback button up top here, and maybe some other
   // stuff.
-  return <Wizard pages={Pages}/>;
+  const imgSrc = browser.extension.getURL("/images/lockie_v1.png");
```

Comment:
I feel like we can make Webpack import images and manage everything for us, but I haven't really looked into that yet...

Reply:
I'd feel more confident about it if I knew what the `data:` URL breakpoint was, and how it interacts with `moz-extension://`.